packages/hardhat/contracts/strategy/CrabMigration.sol

```diff
@@ -146,17 +150,33 @@ import {StrategyMath} from "./base/StrategyMath.sol";
          crabV2.transfer(msg.sender, amountV2ToTransfer);
      }
 
-     /** 
+      /** 
       * @notice allows users to claim crabV2 shares and flash withdraw from crabV2
```

Comment:
Is there a reason to have this instead of just claiming and then withdrawing?

Reply:
The main and only reason is to avoid 2 tx.